doc/classes/@GDScript.xml

```diff
@@ -348,6 +394,15 @@
 			<argument index="1" name="funcname" type="String">
 			</argument>
 			<description>
+				Returns a reference to the specified function 'funcname' in object 'instance'.
+				[codeblock]
+				a = funcref(self, "foo")
+				a.call_func()
+
+				func foo():
```

Comment:
Suggestion: define the function before you assign it to the variable? So it read linearly